CONTRIBUTING.md

```diff
@@ -139,6 +139,25 @@ This package has a `src/agoradatatools/etl/transform` submodule.  This folder ho
     - Use `pytest.mark.parameterize` to loop through multiple datasets in a single test.
     - The class `TestTransformGenesBiodomains` can be used as an example for future tests contibuted.
 
+### Great Expectations
+
+This package uses [Great Expectations](https://greatexpectations.io/) to validate output data.  The `great_expectations` folder houses our file system data context and Great Expectations-specific configuration files. Eventually, our goal is for each `agora-data-tools` dataset to be convered by an expectation suite. To add data validation for more datasets, follow these steps:
+
+1. Create a new expectation suite by defining the expectations for the new dataset in a Jupyter Notebook inside the `gx_suite_definitions` folder. Use `metabolomics.ipynb` as an example.
```

Comment:
I agree with Phil here, get the best of both worlds by having a python script and calling the python script within the jupyter notebook.

Reply:
I agree with the point about interpreting diffs and committing changes being weird in a Jupyter Notebook. But, if I am going to put all of the logic into a Python script anyway, shouldn't I just have people execute the Python script to generate the expectation suite? It seems roundabout to import the logic from a script into Jupyter Notebooks when the main reason I thought a notebook would be a good idea is so that you could test new expectations in a more line-by-line way.

Reply:
After discussion with @thomasyu888: We are going to try it the Jupyter Notebook way for now, but if maintaining Jupyter Notebooks becomes a problem we can reassess and shift to using scripts.